src/libraries/System.Reflection.Emit/src/System/Reflection/Emit/MethodBuilderImpl.cs

```diff
@@ -65,6 +76,11 @@ internal sealed class MethodBuilderImpl : MethodBuilder
             _initLocals = true;
         }
 
+        internal void CreateDllData(string dllName, string entryName, CallingConvention nativeCallConv, CharSet nativeCharSet)
+        {
+            _dllImportData = DllImportData.CreateDllImportData(dllName, entryName, nativeCallConv, nativeCharSet);
```

Comment:
```suggestion
            _dllImportData = DllImportData.Create(dllName, entryName, nativeCallConv, nativeCharSet);
```

I would change API to be `Create()` and then name this function `CreateDllImportData()`.